cuppa/src/main/java/com/hartwig/hmftools/cup/cli/ShellCommand.java

```diff
@@ -18,7 +18,7 @@ public abstract class ShellCommand
 
     @Nullable
     private Level mLogLevel = Level.INFO;
-    private boolean mShowCommand = false;
+    private boolean mShowCommand = true;
```

Comment:
Previously, I set `mShowCommand` to true in certain `ShellCommand` calls where i thought it would be important.

Having `mShowCommand=true` not may be overly verbose:
```
12:34:00.472 [main] [INFO ] Cuppa version 2.2.1
12:34:00.474 [main] [INFO ] Creating output dir: /var/folders/r7/3t6mpsqs2ysfqk1pslgsmy8h0000gn/T/cuppa_output
12:34:00.677 [main] [INFO ] Writing data to: /var/folders/r7/3t6mpsqs2ysfqk1pslgsmy8h0000gn/T//cuppa_output//TUMOR_SAMPLE.cuppa_data.tsv.gz
12:34:00.690 [main] [INFO ] Cuppa data extraction complete, mins(0.004)
12:34:00.691 [main] [INFO ] Running command: [/bin/bash -c /Users/lnguyen/.pyenv/versions/3.9.4/envs/pycuppa_venv/bin/python -m pip list --format=freeze --disable-pip-version-check --exclude-editable]
12:34:00.887 [main] [INFO ] Command [/bin/bash -c /Users/lnguyen/.pyenv/versions/3.9.4/envs/pycuppa_venv/bin/python -m pip list --format=freeze --disable-pip-version-check --exclude-editable] complete
12:34:00.888 [main] [INFO ] Running command: [/bin/bash -c /Users/lnguyen/.pyenv/versions/3.9.4/envs/pycuppa_venv/bin/python -m pip list --format=freeze --disable-pip-version-check --editable]
12:34:01.071 [main] [INFO ] Command [/bin/bash -c /Users/lnguyen/.pyenv/versions/3.9.4/envs/pycuppa_venv/bin/python -m pip list --format=freeze --disable-pip-version-check --editable] complete
12:34:01.071 [main] [INFO ] Predicting using command: /bin/bash -c /Users/lnguyen/.pyenv/versions/3.9.4/envs/pycuppa_venv/bin/python -m cuppa.predict --classifier_path /Users/lnguyen/Hartwig/cloud_source_repos/common-resources-public/cuppa/37/cuppa_classifier.37.pickle.gz --features_path /var/folders/r7/3t6mpsqs2ysfqk1pslgsmy8h0000gn/T//cuppa_output//TUMOR_SAMPLE.cuppa_data.tsv.gz --output_dir /var/folders/r7/3t6mpsqs2ysfqk1pslgsmy8h0000gn/T//cuppa_output/ --sample_id TUMOR_SAMPLE --log_format '[python] %(levelname)s %(name)s | %(message)s'
12:34:01.071 [main] [INFO ] Running command: [/bin/bash -c /Users/lnguyen/.pyenv/versions/3.9.4/envs/pycuppa_venv/bin/python -m cuppa.predict --classifier_path /Users/lnguyen/Hartwig/cloud_source_repos/common-resources-public/cuppa/37/cuppa_classifier.37.pickle.gz --features_path /var/folders/r7/3t6mpsqs2ysfqk1pslgsmy8h0000gn/T//cuppa_output//TUMOR_SAMPLE.cuppa_data.tsv.gz --output_dir /var/folders/r7/3t6mpsqs2ysfqk1pslgsmy8h0000gn/T//cuppa_output/ --sample_id TUMOR_SAMPLE --log_format '[python] %(levelname)s %(name)s | %(message)s']
12:34:09.117 [main] [INFO ] Command [/bin/bash -c /Users/lnguyen/.pyenv/versions/3.9.4/envs/pycuppa_venv/bin/python -m cuppa.predict --classifier_path /Users/lnguyen/Hartwig/cloud_source_repos/common-resources-public/cuppa/37/cuppa_classifier.37.pickle.gz --features_path /var/folders/r7/3t6mpsqs2ysfqk1pslgsmy8h0000gn/T//cuppa_output//TUMOR_SAMPLE.cuppa_data.tsv.gz --output_dir /var/folders/r7/3t6mpsqs2ysfqk1pslgsmy8h0000gn/T//cuppa_output/ --sample_id TUMOR_SAMPLE --log_format '[python] %(levelname)s %(name)s | %(message)s'] complete
...
```

Compare when `mShowCommand=false`:
```
12:32:12.246 [main] [INFO ] Cuppa version 2.2.1
12:32:12.248 [main] [INFO ] Creating output dir: /var/folders/r7/3t6mpsqs2ysfqk1pslgsmy8h0000gn/T/cuppa_output
12:32:12.451 [main] [INFO ] Writing data to: /var/folders/r7/3t6mpsqs2ysfqk1pslgsmy8h0000gn/T//cuppa_output//TUMOR_SAMPLE.cuppa_data.tsv.gz
12:32:12.465 [main] [INFO ] Cuppa data extraction complete, mins(0.004)
12:32:12.655 [main] [INFO ] Command [/bin/bash -c /Users/lnguyen/.pyenv/versions/3.9.4/envs/pycuppa_venv/bin/python -m pip list --format=freeze --disable-pip-version-check --exclude-editable] complete
12:32:12.836 [main] [INFO ] Command [/bin/bash -c /Users/lnguyen/.pyenv/versions/3.9.4/envs/pycuppa_venv/bin/python -m pip list --format=freeze --disable-pip-version-check --editable] complete
12:32:12.836 [main] [INFO ] Predicting using command: /bin/bash -c /Users/lnguyen/.pyenv/versions/3.9.4/envs/pycuppa_venv/bin/python -m cuppa.predict --classifier_path /Users/lnguyen/Hartwig/cloud_source_repos/common-resources-public/cuppa/37/cuppa_classifier.37.pickle.gz --features_path /var/folders/r7/3t6mpsqs2ysfqk1pslgsmy8h0000gn/T//cuppa_output//TUMOR_SAMPLE.cuppa_data.tsv.gz --output_dir /var/folders/r7/3t6mpsqs2ysfqk1pslgsmy8h0000gn/T//cuppa_output/ --sample_id TUMOR_SAMPLE --log_format '[python] %(levelname)s %(name)s | %(message)s'
12:32:20.575 [main] [INFO ] Command [/bin/bash -c /Users/lnguyen/.pyenv/versions/3.9.4/envs/pycuppa_venv/bin/python -m cuppa.predict --classifier_path /Users/lnguyen/Hartwig/cloud_source_repos/common-resources-public/cuppa/37/cuppa_classifier.37.pickle.gz --features_path /var/folders/r7/3t6mpsqs2ysfqk1pslgsmy8h0000gn/T//cuppa_output//TUMOR_SAMPLE.cuppa_data.tsv.gz --output_dir /var/folders/r7/3t6mpsqs2ysfqk1pslgsmy8h0000gn/T//cuppa_output/ --sample_id TUMOR_SAMPLE --log_format '[python] %(levelname)s %(name)s | %(message)s'] complete
...
```